packages/ui/components/NavLink.tsx

```diff
@@ -7,14 +7,24 @@ interface NavLinkProps {
   icon: React.ReactElement;
   href: string;
   closeDrawer: () => void;
+  handleLogout?: () => void;
```

Comment:
You should also call this `handleClick`. A different `NavLink` may wish to have its own click handler passed into it.

Reply:
Okay, I created a `handleClick` function in `Drawer.tsx` and passed it as a prop to all the Navlinks. And for the logout's `handlClick` behavior, I passed a `handleClickLogout` function that calls the handleLogout function from `./auth`